sympy/polys/ring_series.py

```diff
@@ -439,10 +431,10 @@ def _series_inversion1(p, x, prec):
         prec = int(prec)
 
     if zm not in p:
-        raise ValueError('no constant term in series')
+        raise ValueError("No constant term in series")
     if _has_constant_term(p - c, x):
-        raise ValueError('p cannot contain a constant term depending on '
-            'parameters')
+        raise ValueError("p cannot contain a constant term depending on "
+                         "parameters")
```

Comment:
> I started using flake8 to solve the issue. It complained a lot

Yeah, automatic checkers complain a lot.
I guess the best approach to automatic code checkers is to start with 
all checks switched off and explicitly enabling only those that make 
sense for a project.